treeherder/etl/job_loader.py

```diff
@@ -145,6 +148,19 @@ def transform(self, pulse_job):
             platform_src = pulse_job[v] if v in pulse_job else default_platform
             x["job"][k] = self._get_platform(platform_src)
 
+        # add some taskcluster metadata if it's available
+        # currently taskcluster doesn't pass the taskId directly, so we'll
+        # to derive it from the guid
+        try:
+            (decoded_task_id, retry_id) = job_guid.split('/')
+            real_task_id = slugid.encode(uuid.UUID(decoded_task_id))
```

Comment:
I apologize for my ignorance here, but I don't follow why you're converting the ``decoded_task_id`` to a ``UUID`` and then doing a ``slugid.encode``.  Isn't that value already going to be the ``taskId`` we want as a slug?  

I'm deducing from this that it's not.  :)  Would you perhaps add a comment in the code here as to why this is necessary?  Thanks!